src/app/models/Cake.ts

```diff
@@ -0,0 +1,15 @@
+enum CakeStatus {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Enumerator should live in a separate file, maybe named cake-status.enum.ts.

Reply:
Moving into a separate file